project/AppDependencies.scala

```diff
@@ -13,7 +13,9 @@ private object AppDependencies {
     "uk.gov.hmrc"       %% "bootstrap-backend-play-28" % bootstrap,
     "uk.gov.hmrc"       %% "domain"                    % "8.3.0-play-28",
     "uk.gov.hmrc"       %% "reactive-circuit-breaker"  % "4.1.0",
-    "org.typelevel"     %% "cats-core"                 % "2.10.0"
+    "org.typelevel"     %% "cats-core"                 % "2.10.0",
```

Comment:
if we can bump up any other dependencies without much pain (including plugins) can we do this. 
Please note the bump to 8.xx for bootstrap causes alot of pain, so this may need to be a tech debt

Reply:
I understand